lib/masamune/commands/hive.rb

```diff
@@ -114,6 +114,8 @@ def after_execute
       return unless @output
 
       filesystem.move_file_to_file(@buffer.path, @output)
+    ensure
+      File.delete(@buffer.path) if @buffer && @buffer.path && File.exists?(@buffer.path)
```

Comment:
The `@buffer` file can be quite large so it should be manually garbage collected